docker-compose.db.yaml

```
@@ -36,4 +36,4 @@ services:
 
 volumes:
   valkey-data:
-  postgres-data:
+  postgres-data:
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)